packages/expo/src/Notifications/ExponentNotifications.web.ts

```diff
@@ -113,4 +118,92 @@ export default {
   async cancelAllScheduledNotificationsAsync(): Promise<void> {
     this.dismissNotification();
   },
+
+  async getExponentPushTokenAsync(): Promise<string> {
+    if (!Constants.manifest.owner || !Constants.manifest.slug) {
+      throw new CodedError(
+        'ERR_WEB_PUSH_NOTIFICATIONS_MISSING_CONFIG',
```

Comment:
Add an Error Codes section under the method that invokes this in the docs, like https://docs.expo.io/versions/latest/sdk/screen-orientation/#screenorientationlockasyncorientationlock. Also add a table of error codes to the bottom of the docs like: https://docs.expo.io/versions/latest/sdk/screen-orientation/#error-codes-2